### PR TITLE
Pgrete/advection 3d

### PR DESCRIPTION
**Pull request by pgrete**
## PR Summary

For the Kokkosizing I wanted a 3D test problem as baseline so I updated the advection test to 3D.
Also fixed loop patterns (k -> j -> i) , scratch pad dimensions, and a leftover naming mismatch from the graphics refactor.

## PR Checklist

- [x] Code passes cpplint
- [doesn't apply] New features are documented.
- [doesn't apply] Adds a test for any bugs fixed. Adds tests for new features.
- [x] Code is formatted

### CODE REVIEW

File: example/advection/advection_package.cpp [hunk 1]
```diff
@@ -39,6 +39,8 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   pkg->AddParam<>("vx", vx);
   Real vy = pin->GetOrAddReal("Advection", "vy", 1.0);
   pkg->AddParam<>("vy", vy);
+  Real vz = pin->GetOrAddReal("Advection", "vz", 0.5);
```
Comment (jdolence) [inline example/advection/advection_package.cpp:42]: @pgrete just curious, why default to 0.5?  I used 1.0 originally for the other directions to make it transit the unit box exactly once per unit time.

File: example/advection/advection_package.cpp [hunk 2]
```diff
@@ -128,9 +130,9 @@ void SquareIt(Container<Real> &rc) {
   int ke = pmb->ncells3 - 1;
   CellVariable<Real> &qin = rc.Get("one_minus_advected");
   CellVariable<Real> &qout = rc.Get("one_minus_advected_sq");
-  for (int i = is; i <= ie; i++) {
```
Comment (jdolence) [inline example/advection/advection_package.cpp:131]: oops good catch!

File: example/advection/advection_package.cpp (hunk 1 above)
Comment (pgrete) [reply example/advection/advection_package.cpp:42]: I wanted to have sth non-trivial, but I also have no objections changing the default.

**Comment by JoshuaSBrown** [comment]
I also can't find anything to complain about. Nice.

**Comment by Yurlungur** [comment]
Looks ready to me. @pgrete do you want me to pull the trigger?

**Pull request merged**